content/hi/docs/reference/glossary/aggregation-layer.md

```diff
@@ -0,0 +1,20 @@
+---
+title: एकत्रीकरण परत
```

Comment:
```suggestion
title: एग्रीगेशन लेयर
```
I'd prefer seeing transliteration instead of translation here. Aggregation layer is a technical term in this context which may lose its meaning or become difficult to understand when translated as in dictionary.